src/test/java/pages/FavoritesPage.java

```diff
@@ -0,0 +1,16 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class FavoritesPage extends LoginPage {
+
+    //locators
+    By favoritesLocator = By.cssSelector(".playlist.favorites:nth-child(1)");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");